example/django/notifications.py

```diff
@@ -7,6 +7,7 @@
 import django
 import websockets
 
+
```

Comment:
Please avoid gratuitous formatting changes. Also, probably the autoformatter built into the project would revert it at the next commit.